test-cases/longevity/longevity-1TB-3days-authorization-and-tls-ssl.yaml

```diff
@@ -0,0 +1,35 @@
+test_duration: 5020
+prepare_write_cmd: ["cassandra-stress write cl=ALL n=1100200300 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=150 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..1100200300"]
```

Comment:
when returning this job to have 4 loaders, we need to split this command back to have 4 (including the `n=` and the `-pop seq=`

Reply:
done